style.css

```diff
@@ -378,3 +380,224 @@ button:active {
 .form ul {
   margin-right: 84px;
 }
+
+.desktop {
+  display: none;
+}
+
+@media (min-width: 475px) {
+  body {
+    width: 100vw;
+  }
+
+  .footer-wrap {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+}
```

Comment:
- [ ]  You should use media queries for the Desktop breakpoint in the CSS file. (the value for min-width property should be in range of 768px or wider)